### PR TITLE
DM-1624 Loriot Whats New 1016 Update

### PR DESCRIPTION
**Pull request by ashhSAG93**

### CODE REVIEW

File: content/release-10-16-0/whatsnew-10-16-0.md [hunk 1]
```diff
@@ -5,3 +5,24 @@ layout: bundle
 ---
 
 Release 10.16.0 includes the following new features or major feature enhancements.
+
+### Multiple LNS Connections for Loriot
```
Comment (BeateRixen) [inline content/release-10-16-0/whatsnew-10-16-0.md:9]:
```suggestion
### Multiple LNS connections for Loriot
```

File: content/release-10-16-0/whatsnew-10-16-0.md [hunk 2]
```diff
@@ -5,3 +5,24 @@ layout: bundle
...
+
+The Loriot agent now supports the configuration of multiple connections per tenant. One connection can be associated with multiple devices. Each connection can be updated and deleted.
+
+![All devices](/images/release-notes/multiple_lns_connection_loriot.png)
+
+For details, refer to [Loriot Multiple LNS Connection](https://cumulocity.com/guides/protocol-integration/lora-loriot/#configure-loriot-credentials-cumulocity)
```
Comment (BeateRixen) [inline content/release-10-16-0/whatsnew-10-16-0.md:15]:
```suggestion
For details, refer to [Device registration via Cumulocity IoT](https://cumulocity.com/guides/protocol-integration/lora-loriot/#configure-loriot-credentials-cumulocity).
```

File: content/release-10-16-0/whatsnew-10-16-0.md [hunk 3]
```diff
@@ -5,3 +5,24 @@ layout: bundle
...
+
+### Support for Loriot Device Registration
```
Comment (BeateRixen) [inline content/release-10-16-0/whatsnew-10-16-0.md:17]:
```suggestion
### Support for Loriot device registration
```

File: content/release-10-16-0/whatsnew-10-16-0.md [hunk 4]
```diff
@@ -5,3 +5,24 @@ layout: bundle
...
+
+Using the Loriot LNS Connections created in the **Connectivity** page in **Settings** under **Administration**, devices can now also be registered using the **Device Registration** page.
```
Comment (BeateRixen) [inline content/release-10-16-0/whatsnew-10-16-0.md:19]:
```suggestion
Using the Loriot LNS connections created in the **Connectivity** page in **Settings** under **Administration**, devices can now also be registered using the **Device Registration** page. 
```

File: content/release-10-16-0/whatsnew-10-16-0.md [hunk 5]
```diff
@@ -5,3 +5,24 @@ layout: bundle
...
+
+For details, refer to [Loriot Multiple LNS Connection](https://cumulocity.com/guides/protocol-integration/lora-loriot/#configure-loriot-credentials-cumulocity)
+
+### Support for Loriot Device Registration
+
+Using the Loriot LNS Connections created in the **Connectivity** page in **Settings** under **Administration**, devices can now also be registered using the **Device Registration** page. 
+
+![Device Registration](/images/release-notes/loriot-registration.png)
+
+For Details refer to [Loriot Device Registration](https://cumulocity.com/guides/protocol-integration/lora-loriot/#configure-loriot-credentials-cumulocity)
```
Comment (BeateRixen) [inline content/release-10-16-0/whatsnew-10-16-0.md:23]:
```suggestion
For details refer to [Device registration via Cumulocity IoT](https://cumulocity.com/guides/protocol-integration/lora-loriot/#configure-loriot-credentials-cumulocity).
```

File: content/release-10-16-0/whatsnew-10-16-0.md [hunk 6]
```diff
@@ -5,3 +5,24 @@ layout: bundle
...
+
+For the devices registered with the Loriot agent, downlink operations can be sent from the **Shell** tab
```
Comment (BeateRixen) [inline content/release-10-16-0/whatsnew-10-16-0.md:25]:
```suggestion
For the devices registered with the Loriot agent, downlink operations can be sent from the **Shell** tab.
```

File: content/release-10-16-0/whatsnew-10-16-0.md [hunk 7]
```diff
@@ -5,3 +5,24 @@ layout: bundle
...
+
+For Details refer to [Loriot Downlink Operations](https://cumulocity.com/guides/protocol-integration/lora-loriot/#operations-loriot)
```
Comment (BeateRixen) [inline content/release-10-16-0/whatsnew-10-16-0.md:27]:
```suggestion
For details refer to [Sending operations](https://cumulocity.com/guides/protocol-integration/lora-loriot/#operations-loriot).
```

Comment (BeateRixen) [reply content/release-10-16-0/whatsnew-10-16-0.md:27]: @ashhSAG93 Please make sure to use the exact headings in links.

Comment (ashhSAG93) [reply content/release-10-16-0/whatsnew-10-16-0.md:27]: Sure @BeateRixen we will check the links and make the changes

**Review by BeateRixen — APPROVED** [review]

**Review by miryalarahul — APPROVED** [review]

**Pull request merged**